pkg/cmd/admin/policy/reconcile_sccs_test.go

```diff
@@ -41,70 +41,105 @@ func TestComputeDefinitions(t *testing.T) {
 	diffFSGroup := goodSCC()
 	diffFSGroup.FSGroup.Type = kapi.FSGroupStrategyMustRunAs
 
+	diffLabels := goodSCC()
+	diffLabels.Labels = map[string]string{
+		"foo": "bar",
+	}
+
+	diffAnnotations := goodSCC()
+	diffAnnotations.Annotations = map[string]string{
+		"status": "my bar",
+	}
+
 	tests := map[string]struct {
-		expected    kapi.SecurityContextConstraints
-		actual      kapi.SecurityContextConstraints
-		needsUpdate bool
+		expected     kapi.SecurityContextConstraints
+		actual       kapi.SecurityContextConstraints
+		needsUpdate  bool
+		diffPeserved bool
```

Comment:
actually, I'd rather add an `updated kapi.SecurityContextConstraints` field to this test, so we can check the actual values that resulted